react_front_end/src/pages/components/LandingPageInformation.js

```diff
@@ -22,7 +31,41 @@ const LandingPageInformation = () => {
   return (
     <>
       <section className={classes.root}>
-        <ExperimentInformation />
+        <Paper square style={{maxWidth:"850px"}}>
```

Comment:
The copy needs to be split into sizeable chunks of information.

1) It’s Spring 2020 and Coachella, SXSW, the NBA, NHL, MLB and the Tokyo Olympics are cancelled this year.

2) We’re at the height of human technology and innovation, but at the same time facing one of the most devastating viral pandemics in history.

So it reads like a song, poem or story. I'm sure @un7c0rn can advise on how best to split it up into sizeable chunks.

There is a swipable library on NPM (https://www.npmjs.com/package/react-swipeable) use it to swipe the text left and right.